### PR TITLE
Single source asciidoc attributes

### PR DESCRIPTION
**Pull request by rsynek**
Moving quickstarts-related attributes to a single .adoc file.

### Checklist
- [ ] Documentation updated if applicable.
- [ ] Upgrade recipe provided if applicable.

<details>
<summary>
How to replicate CI configuration locally?
</summary>

We do "simple" maven builds, they are just basically maven commands, but just because we have multiple repositories related between them and one change could affect several of those projects by multiple pull requests, we use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.

[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is not only a github-action tool but a CLI one, so in case you posted multiple pull requests related with this change you can easily reproduce the same build by executing it locally. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>specific pull request build</b> please add comment: <b>Jenkins (re)run [optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] tests</b>
* for a <b>full downstream build</b> 
  * for <b>jenkins</b> job: please add comment: <b>Jenkins run fdb</b>
  * for <b>github actions</b> job: add the label `run_fdb`
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
* for a <b>Quarkus LTS check</b> please add comment: <b>Jenkins run LTS</b>
* for a <b>specific Quarkus LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] LTS</b>
* for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
* for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] native</b>
</details>

### CODE REVIEW

File: optaplanner-docs/src/modules/ROOT/pages/quickstart/hello-world/hello-world-quickstart.adoc [hunk 1]
```diff
@@ -4,9 +4,7 @@
 :imagesdir: ../..
 :sectnums:
 :icons: font
-:quickstartsCloneUrl: https://github.com/kiegroup/optaplanner-quickstarts
-:quickstartsArchiveUrl: https://www.optaplanner.org/download/download.html
-:helloWorldJavaQuickstartUrl: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/hello-world
+include::../../_attributes.adoc[]
```
Comment (rsynek) [inline optaplanner-docs/src/modules/ROOT/pages/quickstart/hello-world/hello-world-quickstart.adoc:7]: The file must start with `_`, see https://docs.antora.org/antora/2.3/standard-directories/#hidden-files for an explanation.

Comment (ge0ffrey) [reply optaplanner-docs/src/modules/ROOT/pages/quickstart/hello-world/hello-world-quickstart.adoc:7]: antora doesn't support ignores in their yml config? Not that I mind the _ so much.
good to merge as is.

**Review by ge0ffrey — APPROVED** [review]

File: optaplanner-docs/src/modules/ROOT/pages/_attributes.adoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+:quickstartsCloneUrl: https://github.com/kiegroup/optaplanner-quickstarts
+:quickstartsArchiveUrl: https://www.optaplanner.org/download/download.html
+:helloWorldJavaQuickstartUrl: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/hello-world
+:springBootQuickstartUrl: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/technology/java-spring-boot
+:quarkusQuickstartUrl: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
```
Comment (ge0ffrey) [inline optaplanner-docs/src/modules/ROOT/pages/_attributes.adoc:5]: Can we change these to follow the asciidoc guidelines style?
So quarkus-quickstart-url IIRC.

Comment (ge0ffrey) [reply optaplanner-docs/src/modules/ROOT/pages/_attributes.adoc:5]: meet.google.com/qur-fmqu-wav

Comment (ge0ffrey) [reply optaplanner-docs/src/modules/ROOT/pages/_attributes.adoc:5]: That probably applies to the maven/antora injected attributes too.

Should we mention those here too, in a comment? Or mention in which file(s) to find those?

File: optaplanner-docs/src/modules/ROOT/pages/quickstart/hello-world/hello-world-quickstart.adoc (hunk 1 above)
Comment (rsynek) [reply optaplanner-docs/src/modules/ROOT/pages/quickstart/hello-world/hello-world-quickstart.adoc:7]: I haven't found such an option and would rather avoid it anyway; imagine maintaining a list of paths in a .yml file

**Pull request merged**